examples/svg/readme.md

```diff
@@ -0,0 +1,22 @@
+# Tungsten.js + Backbone TodoMVC Example
```

Comment:
This looks wrong... entirely possible this is on me, but this should probably get deleted